<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpzioniDialog</class>
 <widget class="QDialog" name="OpzioniDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Impostazioni generali</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Report dir</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="reportDirLE"/>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Bin dir</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Backup dir</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Biblio dir</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="binDirLE"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="backupDirLE"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="biblioDirLE"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Collezione</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Links</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Biblioteca</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Contenitori</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Contatti</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="collezioneLE"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="linksLE"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="bibliotecaLE"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="contenitoriLE"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="contattiLE"/>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_7">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButton_8">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pushButton_9">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
